ff2mpv

```diff
@@ -4,7 +4,14 @@
 require "json"
 
 len = STDIN.read(4).unpack1("L")
-url = JSON.parse(STDIN.read(len))["url"]
+data = JSON.parse(STDIN.read(len))
+url = data["url"]
+opt = data["opt"]
+
+if opt == "video"
```

Comment:
This is probably too duplicative. Instead of calling `spawn` separately, let's maybe do something like this:

```ruby
args = %w[--no-terminal]

unless opt["video"]
  args << "--no-video"
end

pid = spawn "mpv", *args, "--", url, in: :close, out: "/dev/null", err: "/dev/null"
```

...and similarly for the Python client.